autopilot:
    ros__parameters:
        x_dot_P: 0.5
        x_dot_I: 0.0
        x_dot_D: 0.05
        y_dot_P: 0.5
        y_dot_I: 0.0
        y_dot_D: 0.05
        z_dot_P: 0.4
        z_dot_I: 0.25
        z_dot_D: 0.1
        north_P: 1.0
        north_I: 0.1
        north_D: 0.35
        east_P: 1.0
        east_I: 0.1
        east_D: 0.2
        down_P: 1.0
        down_I: 0.0
        down_D: 0.0
        psi_P: 2.0
        psi_I: 0.0
        psi_D: 0.0

        kp_phi: 0.2
        ki_phi: 0.0
        kd_phi: -0.2
        kp_theta: 0.2
        ki_theta: 0.0
        kd_theta: -0.2
        kp_psi: 0.2
        ki_psi: 0.002
        kd_psi: -0.2
        kp_u_n: 0.2
        ki_u_n: 0.002
        kd_u_n: -0.2
        max_u_n_dot: 5.0
        kp_u_e: 0.2
        ki_u_e: 0.002
        kd_u_e: -0.2
        max_u_e_dot: 5.0
        kp_u_d: 0.2
        ki_u_d: 0.002
        kd_u_d: -0.2
        max_u_d_dot: 5.0

        tau: 0.05
        C_d: 0.0
        gravity: 9.81
        mass: 2.0
        equilibrium_throttle: 0.3
        max_roll: 0.196
        max_pitch: 0.196
        max_yaw_rate: 0.785
        max_throttle: 0.85
        min_throttle: 0.0
        max_n_dot: 1.5
        max_e_dot: 1.5
        max_d_dot: 0.5
        max_roll_torque: 0.1 
        max_pitch_torque: 0.1
        max_yaw_torque: 0.1  
        max_thrust: 4.0      
        min_altitude: 0.5